src/control/controlindicatortimer.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <QObject>
+#include <QTimer>
+#include <memory>
+
+#include "control/controlobject.h"
+
+namespace mixxx {
+
+/// This class provides two generic indicator controls that change their value
+/// in intervals of 250 milliseconds and 500 milliseconds. These controls are
+/// used by all `ControlIndicator` COs and may also be used to implement custom
+/// blinking buttons (e.g. for Explicit Sync Leader) on controllers. This
```

Comment:
huh?

Reply:
Do you have a suggestion what to change? I think it makes sense to explain what these COs are used for and that they are not "internal", i.e. that they are useful for third party code, too. I was planning to update the Roland DJ-505 to use them, because the blinking of the SUNC button not being in sync (hah! ;-) with the CUE button is annoying.

Reply:
I don't think controller scripts should be implementing custom blinking in general.

Reply:
In general I agree that we should avoid blinking. But in some cases it can be useful. My sync button only has a single color, and in the rare cases I deem it necessary to set a sync leader explicitly, I want to know which deck is sync leader.

Reply:
active loop, playing sampler, key set off etc. all deserve blinking if the user considers that state unusual/remarkable